l3build-install.lua

```diff
@@ -200,17 +199,18 @@ function install_files(target,full,dry_run)
           end
         end
       end
+      local ans = { }
       for _,glob in pairs(include) do
         for file,_ in pairs(tree(dir,glob)) do
           if not excludelist[file] then
-            insert(includelist, file)
+            insert(ans, file)
```

Comment:
1. YMMV but I would use `res`, `result` or `ret` here. (res and result are already used so they are more consistent, ret shows that it is a generic name for a *ret*urn value. For me `ans` sounds more like a result which is presented to the user.)
  2. The combination of using the more generic name `create_list` and the extremely generic return value name `ans` makes it harder to see on the first glance what this list contains. (Arguably it's still clearer than the old version which claimed the opposite of what it did :laughing:) Maybe `create_filelist`?